UnityProject/Assets/Scripts/Objects/Doors/DoorMasterController.cs

```diff
@@ -112,6 +119,21 @@ private void Awake()
 			apc = GetComponent<APCPoweredDevice>();
 			doorAnimator = GetComponent<DoorAnimatorV2>();
 			doorAnimator.AnimationFinished += OnAnimationFinished;
+			LoadManager.RegisterActionDelayed(DelayedRegister, 2);
+		}
+
+		public void DelayedRegister()
+		{
+			HackingProcessBase.RegisterPort(EmptyFunction, this.GetType()); //So there is an option that is nothing
+			HackingProcessBase.RegisterPort(TryForceClose, this.GetType());
+			HackingProcessBase.RegisterPort(tryBump, this.GetType());
+			HackingProcessBase.RegisterPort(TryClose, this.GetType());
+
+		}
+
+		public void EmptyFunction()
+		{
+
 		}
```

Comment:
?

Reply:
is dummy function so one of the cables doesn't do anything

Reply:
Cool